src/client/auth/auth.proto

```diff
@@ -372,6 +372,25 @@ message SetACLRequest {
 
 message SetACLResponse {}
 
+//////////////////////////////
+//// OIDC Data Structures ////
+//////////////////////////////
+
+// SessionInfo stores information associated with one OIDC authentication flow.
+// Flows are identified with a 'state' token generated by 'GetOIDCLogin'. The
+// 'oidc-auth-flows' etcd collection maps an OIDC state token (key, 30-character
+// string) to collection under and passed via 'Authenticate' and
+// /authorization-code/callback. Instances of 'sessionInfo' are stothe key value
+// in 'stateInfoMap' under which a given instance of 'sessionInfo' is stored) so
+// that that /callback can pass the
+message SessionInfo {
+  // nonce is used by Authenticate() to validate session continuity with the IdP
+  // when exchanging an authorization code for a user's user info.
+  string nonce = 1;
+  // id_token contains a user's OIDC ID token, which can be used to get their
+  // user info.
+  string id_token = 2 [(gogoproto.customname) = "IDToken"];
```

Comment:
add error field to display any authorization code conversion errors in terminal.
Implement printing errors in `Authorize()`